gammapy/scripts/config/schema.yaml

```diff
@@ -229,36 +279,34 @@ properties:
 #                unit: TeV
 #                interp: log
 #
-    geometry:
-        type: object
-        additionalProperties: false
-        properties:
-            binsz: {type: number}
-            coordsys:
-                type: string
-                enum: [CEL, GAL]
-            proj: {type: string}
-            skydir:
-                type: array
-                items: {type: number}
-                minItems: 2
-                maxItems: 2
-            width:
-                type: array
-                items: {type: number}
-                minItems: 1
-                maxItems: 2
-            offset_max: {type: string}
-            psf_max_radius: {type: string}
-            axes:
-                type: object
-                additionalProperties: false
-                properties:
-                    e_reco:
-                        "$ref": "#/definitions/energy_axis"
-                    e_true:
-                        "$ref": "#/definitions/energy_axis"
-        required: [axes]
+#    geom:
+#        type: object
+#        additionalProperties: false
+#        properties:
+#            binsz: {type: number}
+#            coordsys:
+#                type: string
+#                enum: [CEL, GAL]
+#            proj: {type: string}
+#            skydir:
+#                type: array
+#                items: {type: number}
+#                minItems: 2
+#                maxItems: 2
+#            width:
+#                type: array
+#                items: {type: number}
+#                minItems: 1
+#                maxItems: 2
+#            axes:
+#                type: object
+#                additionalProperties: false
+#                properties:
+#                    e_reco:
+#                        "$ref": "#/definitions/energy_axis"
+#                    e_true:
+#                        "$ref": "#/definitions/energy_axis"
+#        required: [axes]
```

Comment:
better remove? it's starting to be a long file..

Reply:
Done